examples/test/test_bert_cuda_gpu.py

```diff
@@ -23,12 +23,20 @@ def setup():
 @pytest.mark.parametrize("execution_order", ["joint", "pass-by-pass"])
 @pytest.mark.parametrize("system", ["aml_system"])
 @pytest.mark.parametrize("olive_json", ["bert_cuda_gpu.json"])
+@pytest.mark.parametrize("no_engine_and_evaluator", [True, False])
```

Comment:
The test already takes a long time to run. I think it might be better to test different settings of the engine as unit tests? 
And keep the example tests as more like integration tests. Or move this `no_engine_and_evalutor` to a faster running test (without aml) like `bert_ptq_cpu`?

Reply:
yeah, unit test should be more appropriated than integration test.